api.yaml

```diff
@@ -990,6 +978,22 @@ components:
           format: date-time
           example: "2019-06-30T19:45:00+00:00"
           description: Datetime string in format YYYY-MM-DDThh:mm:ss+00:00. Timezone is always UTC
+        pricing:
+          type: object
+          description: |
+            If the product's `provides_pricing = true` then the response MUST specify the prices of all the
+            variants in the reservation. If `provides_pricing = false` then this attribute can be omitted from the response.
+          additionalProperties:
+            type: object
+            properties:
+              amount:
+                description: The price the customer will pay for this variant.
```

Comment:
The **unite** price the customer will pay for **a single** ticket of this variant.